Cargo.toml

```diff
@@ -0,0 +1,26 @@
+
+
+[workspace]
+# We want to use v2 of the cargo dependency resolver.
+resolver = "2"
```

Comment:
Great ! yeah i think there was some discussions about making this the default IIRC.